src/Rings/mpoly-graded.jl

```diff
@@ -156,7 +158,14 @@ Multivariate polynomial ring in 3 variables over QQ graded by
   x -> [1]
   y -> [1]
   z -> [1]
+```
 
+Grading an already graded polynomial ring is not supported.
+To produce a new ring with different grading, you need to first
+call `forget_grading` and then `grade` the result.
+```jldoctest grade-ex
+julia> grade(S)
+ERROR: ArgumentError: cannot grade polynomial ring which is already decorated
 ```
```

Comment:
````suggestion
```jldoctest grade-ex
julia> grade(S)
ERROR: ArgumentError: cannot grade polynomial ring which is already decorated
```
To produce a new ring with different grading, you need to first
call `forget_grading` and then `grade` the result.
````